.github/workflows/docker.yml

```diff
@@ -0,0 +1,62 @@
+name: Docker image
+
+#on:
```

Comment:
Could this be removed?

Reply:
the intent was to keep it a a reference should we decide to provide images of our releases only. I could clarify that with another comment?